### PR TITLE
Added Tina fields setting the SEO for both docs and blogs

### PR DESCRIPTION
**Pull request by isaaclombardssw**
### General Contributing:

- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: tina/customTinaFormFields/textInputWithCount.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import React from "react";
+import { wrapFieldsWithMeta } from "tinacms";
+
+export const TextInputWithCount = (max: number, isTextArea: boolean = false) =>
+  wrapFieldsWithMeta(({ input }) => (
+    <div className="flex flex-col gap-2">
+      {isTextArea ? (
+        <textarea
+          // eslint-disable-next-line tailwindcss/no-custom-classname
```
Comment (joshbermanssw) [inline tina/customTinaFormFields/textInputWithCount.tsx:11]: why are we disabling the eslint here? add comment pls

**Review by joshbermanssw — APPROVED** [review]
LGTM just one comment pls

File: tina/customTinaFormFields/textInputWithCount.tsx (hunk 1 above)
Comment (isaaclombardssw) [reply tina/customTinaFormFields/textInputWithCount.tsx:11]: I'll investigate

Comment (isaaclombardssw) [reply tina/customTinaFormFields/textInputWithCount.tsx]: Removed 👌

**Pull request merged**